hmac_authentication/hmacauth.py

```diff
@@ -1,57 +1,65 @@
 # -*- coding: utf-8 -*-
 
 from hmac_authentication import exceptions
+
 import base64
+import collections
+import enum
```

Comment:
Is the actual package name `backports`? https://pypi.python.org/pypi/backports/1.0

I couldn't find `backported`: https://pypi.python.org/pypi?%3Aaction=search&term=backported&submit=search